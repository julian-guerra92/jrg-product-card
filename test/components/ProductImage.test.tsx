import React from 'react';
import render from 'react-test-renderer'
import { ProductCard } from '../../src/components/ProductCard';
import { ProductImage } from '../../src/components/ProductImage';
import { product2 } from '../data/productos';

describe('Pruebas en el ProductImage', () => {

   test('Debe de mostrar el componenete correctamente con la imagen correspondiente', () => {
      const wraper = render.create(
         <ProductImage img={product2.img} />
      )
      expect(wraper.toJSON()).toMatchSnapshot();
   })

   test('Debe de mostrar el componenete con la imagen del producto', () => {
      const wraper = render.create(
         <ProductCard product={product2}>
            {
               () => (
                  <ProductImage />
               )
            }
         </ProductCard>
      )
      expect(wraper.toJSON()).toMatchSnapshot();
   })


})
