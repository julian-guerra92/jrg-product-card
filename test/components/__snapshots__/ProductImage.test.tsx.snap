// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el ProductImage Debe de mostrar el componenete con la imagen del producto 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`Pruebas en el ProductImage Debe de mostrar el componenete correctamente con la imagen correspondiente 1`] = `
<img
  alt="Product"
  className="productImg undefined"
  src="./coffee-mug.png"
/>
`;
